st2tests/st2tests/pack_resource.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import inspect
+
+from unittest2 import TestCase
+
+__all__ = [
+    'BasePackResourceTestCase'
+]
+
+
+class BasePackResourceTestCase(TestCase):
+    """
+    Base test class for all the pack resource test classes.
+
+    Contains some utility methods for loading fixtures from disk, etc.
+    """
+
+    def get_fixture_content(self, fixture_path):
```

Comment:
I think it's fine if user wants to store fixture in a sub-directory (e.g. `tests/fixtures/api_responses/my_fixture.json` or similar)